lib/galaxy/dependencies/pipfiles/default/pinned-requirements.txt

```diff
@@ -78,6 +78,7 @@ jmespath==0.9.4
 jsonpatch==1.25
 jsonpointer==2.0
 jsonschema==3.2.0
+keras==2.2.4
```

Comment:
Can we swap this and tensorflow to conditional requirements based on `enable_tool_recommendations` in the config?

Reply:
Fixed it. Thanks a lot!